src/components/Pagination/Pagination.js

```diff
@@ -4,8 +4,9 @@ import classNames from "classnames";
 
 function Pagination({ currentPage, handlePageChange, totalItem, limit }) {
   const pageIndexes = [currentPage];
+  const paginationRange = 2;
```

Comment:
Can we get the total of pages by the total of items?

Reply:
Actually, we can, but i do this for the limit of page indexes. if we have total of 6 pages and our current page is 4 so the page indexes is 2,3,4,5,6

Reply:
We should make sure that it always works without modifying in the future just to extend the number of pages. For your case, we can have the next & last buttons or re-do the pagination.